pkg/models/secrets_format.go

```diff
@@ -21,30 +21,32 @@ type SecretsFormat int
 // the source of the value
 const (
 	JSON SecretsFormat = iota
+	DOTNET_JSON
 	ENV
 	YAML
 	DOCKER
-	ENV_NO_FILE
+	ENV_NO_QUOTES
```

Comment:
Can you move this rename into a separate commit? You'll want to prefix it with `chore: ` so it doesn't show in the changelog.